WinUIGallery/Samples/ControlPages/ContentIslandPage.xaml.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Numerics;
+using Microsoft.UI.Composition;
+using Microsoft.UI.Content;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Hosting;
+using Microsoft.UI.Xaml.Shapes;
+
+namespace WinUIGallery.ControlPages;
+
+public sealed partial class ContentIslandPage : Page
+{
+    public ContentIslandPage()
+    {
+        this.InitializeComponent();
+    }
+
+    int idx = 0;
+
+    Rectangle GetNextHostElement()
+    {
+        if (idx < _rectanglePanel.Children.Count)
+        {
+            return ((Rectangle)_rectanglePanel.Children[idx++]);
+        }
+
+        return null;
+    }
+
+    public async void LoadModel()
+    {
+        ContentIsland parentIsland = this.XamlRoot.TryGetContentIsland();
```

Comment:
```suggestion
        ContentIsland parentIsland = this.XamlRoot.ContentIsland;
```
It looks like this has changed in WASDK 1.7.
![image](https://github.com/user-attachments/assets/671ece5e-c671-4743-b387-a5a9b86e3958)